src/lender/coordinator.sol

```diff
@@ -30,30 +17,30 @@ interface ReserveLike {
     function balance() external;
 }
 
-contract AssessorLike is FixedPoint {
+abstract contract AssessorLike is FixedPoint {
```

Comment:
I think here it is fine to still use an abstract contract otherwise it might be tricky with the Fixed27 type.

It is easier to read if we can use them in the `Like` definitions.